example/libclang-example/generated_bindings.dart

```diff
@@ -10432,6 +10434,11 @@ typedef CXInclusionVisitor = ffi.Pointer<
             ffi.Pointer<CXSourceLocation> inclusion_stack,
             ffi.UnsignedInt include_len,
             CXClientData client_data)>>;
+typedef CXInclusionVisitor_function = ffi.Void Function(
```

Comment:
Naming-wise, it feels like this one should be without `_function` and the other one should be renamed `_pointer`. Do you know if we have a lot of uses of these typedefs, if it would be worth the breaking change?

Reply:
Not sure, but we could avoid a breaking change by putting this behind a flag. Eg, add a flag called `use-function-typedefs`, and if it's true we add `_pointer` to the existing definition and then define the function definition with the current name. WDYT?

Reply:
Let's just merge as is.